dev/CommandBarFlyout/CommandBarFlyout_themeresources.xaml

```diff
@@ -304,7 +304,7 @@
                                 HorizontalAlignment="Stretch"
                                 VerticalAlignment="Center"
                                 Margin="12,0,12,0"
-                                Padding="0,5,0,7"
+                                Padding="0,6,0,7"
```

Comment:
Why those padding changes here and below?

Reply:
Because I accidentally reused an old branch.. that change was already merged in, but github was getting confused.